lib/Jit/EEMemoryManager.cpp

```diff
@@ -74,15 +75,65 @@ bool EEMemoryManager::finalizeMemory(std::string *ErrMsg) {
   return true;
 }
 
+static size_t getXdataSize(const uint8_t *PdataPtr) {
```

Comment:
Naming nit: `PdataPtr` seems unfortunate because it sounds like it is referring to the .pdata section. Since the .pdata points at the .xdata doing things this way is not all that implausible. But it's not pointing there.

Also given that this is a file static it needs doxygen markup here.